src/android/Zoom.java

```diff
@@ -228,6 +228,7 @@ public class Zoom extends CordovaPlugin implements ZoomSDKAuthenticationListener
     private final Handler callIgnoredHandler = new Handler();
     private static Zoom mInstance = null;
 
+    public AlertDialog messageDialog;
```

Comment:
We can change access modifier to Private

Reply:
Done.